### PR TITLE
document default payment term in paymentTerms.list

### PR DESCRIPTION
**Pull request by joaoinacio**

### CODE REVIEW

**Review by mrtus — DISMISSED** [review]
💯

**Comment by tiagobastos** [comment]
✔️ Manually checked

File: apiary.apib [hunk 1]
```diff
@@ -4421,6 +4422,8 @@ Get a list of available payment terms.
             + (object)
                 + id: `c93ddb52-0af8-47d9-8551-441435be66a7` (string)
                 + Include PaymentTerm
+        + meta (object)
+            + `default`: `c93ddb52-0af8-47d9-8551-441435be66a7` (string)
```
Comment (mrtus) [inline apiary.apib:4426]: Let's make sure we align this property structure with all future ones we'd do, e.g. dealPipelines cc @gdegirmenci

**Review by Jdww — DISMISSED** [review]

**Review by mrtus — APPROVED** [review]

**Review by Jdww — APPROVED** [review]

File: apiary.apib (hunk 1 above)
Comment (mrtus) [inline apiary.apib:4426]: I've actually been having some doubts for the past couple of days about the naming, should we include what the default is about?

```
default_payment_term: `c93ddb52-0af8-47d9-8551-441435be66a7`
```


What if we introduce more "default" a-like values?

Comment (mrtus) [reply apiary.apib:4426]: Or apply more structure through
```
default: {
  id: c93ddb52-0af8-47d9-8551-441435be66a7,
  type: paymentTerm
}
```

Comment (mrtus) [reply apiary.apib:4426]: Or
```
default_payment_term: {
  id: c93ddb52-0af8-47d9-8551-441435be66a7,
  type: paymentTerm
}
```

Comment (Miljar) [reply apiary.apib:4426]: I'm actually fine with the original. My reasoning: We're on a `.list` endpoint and will return a single list (maybe paginated) of items of a single type. So `default` is for me clearly the default item from that list.

**Review by tiagobastos — APPROVED** [review]

File: apiary.apib (hunk 1 above)
Comment (mrtus) [reply apiary.apib:4426]: We decided on keeping as is.

**Pull request merged**